### ISSUE TITLE
Design changes to opendatahub.com

### ISSUE THREAD

**Issue by henri-egger** [noi-techpark/opendatahub-website#130]
The changes in the [linked](https://noibz.sharepoint.com/:p:/s/UNITshrdTech-TransferDigital/EZIxbFBLF5xIu8RL_z2xZUcBMOuP4jgMU_TQ832YT7KaRA?e=vBdfGh) PowerPoint should implemented in a separate branch of the opendatahub website.

**Comment by ohnewein** [comment]
Hu @henri-egger and @emilylouisesimonis , testing the https://opendatahub.testingmachine.eu/ I noticed that the menu sub-entries, like "Data sharing" under "Services" points to the services page with the anchor #sm-tag-data-sharing instead of to the "Data sharing" page: https://opendatahub.testingmachine.eu/data-sharing/ 

I would have expected to get directly in the dedicated page instead of an anchor in the parent page. Especially if I select the first entry "Data access" on my mobile I got confused and thought the menu doesn't work, at first.

Is this a technical limitation or a design/content decision?
